### PR TITLE
Create PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by rhyshort**
## What
Create initial PULL_REQUEST_TEMPLATE, this makes it clear to potential contributors what we expect and save the admin of asking for CLA's to be signed, tests to be added etc.

## Reviewers

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+Thanks for your hard work, please ensure all items are complete before opening.
+
+- [ ] Make sure you have signed the CLA as per the instrcutions in [CONTRIBUTING.md](https://github.com/cloudant/java-cloudant/blob/master/CONTRIBUTING.md#contributor-license-agreement)
```
Comment (ricellis) [inline .github/PULL_REQUEST_TEMPLATE.md]: I would make this a numbered list:

1. You have signed ...

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+- [ ] If you Changes include any code changes, make sure tests are present
```
Comment (ricellis) [inline .github/PULL_REQUEST_TEMPLATE.md]: 2. You have added tests for any code changes

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+- [ ] Follow the template below:
```
Comment (ricellis) [inline .github/PULL_REQUEST_TEMPLATE.md]: You have completed the PR template below:

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+## What
+
+What was changed, eg Added support for querying views.
```
Comment (ricellis) [inline .github/PULL_REQUEST_TEMPLATE.md]: e.g.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+## How
+
+How the change was implemented and reasoning behind it, eg. Added ViewRequestBuilder API to query views, this allows for
```
Comment (ricellis) [inline .github/PULL_REQUEST_TEMPLATE.md]: e.g.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (rhyshort) [inline .github/PULL_REQUEST_TEMPLATE.md]: done in 7037acb

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (rhyshort) [inline .github/PULL_REQUEST_TEMPLATE.md]: done in 7037acb

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (rhyshort) [inline .github/PULL_REQUEST_TEMPLATE.md]: done in 7037acb

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 4 above)
Comment (rhyshort) [inline .github/PULL_REQUEST_TEMPLATE.md]: done in 7037acb

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 5 above)
Comment (rhyshort) [inline .github/PULL_REQUEST_TEMPLATE.md]: done in 7037acb

**Comment by ricellis** [comment]
+1

**Pull request merged**